pymtl3/stdlib/rtl/enrdy_queues.py

```diff
@@ -44,7 +44,7 @@ def construct( s, Type ):
 
     s.buffer  = RegEn( Type )( in_ = s.enq.msg )
 
-    s.full = Reg( Bits1 )
+    s.full = RegRst( Bits1, reset_value = Bits1(0) )
```

Comment:
you might only need 0 because RegRst does something like Type(reset_value). I was thinking about a register that register a BitStruct ... the current approach won't work.